src/content/docs/develop/sidecar.mdx

```diff
@@ -131,6 +131,39 @@ You can place this code inside a Tauri command to easily pass the AppHandle or y
 
 ## Running it from JavaScript
 
+When running the sidecar, Tauri requires you to give the sidecar permission to run the `execute` or `spawn` method on the child process. To grant this permission, go to the file `<PROJECT ROOT>/src-tauri/capabilities/default.json` and add the section below to the permissions array. Don't forget to name your sidecar according to the relative path mentioned earlier.
+
+```json title="src-tauri/capabilities/default.json" ins={11-20}
+{
+   "permissions": [
+      "core:path:default",
+      "core:event:default",
+      "core:window:default",
+      "core:app:default",
+      "core:resources:default",
+      "core:menu:default",
+      "core:tray:default",
+      "store:default",
```

Comment:
```suggestion
      "core:default",
```
ah i forgot, i think we can simplify it to just this (same for the old example further down)